views/index.dt

```diff
@@ -0,0 +1,21 @@
+!!! 5
```

Comment:
Oh it's mentioned [here](https://github.com/rejectedsoftware/diet-ng/blob/master/SPEC.md#doctype-specifications)

> Legacy syntax: !!! ... will be transformed to doctype ...